index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { Octokit, App } = require("octokit");
+
+async function run() {
+    const githubToken = core.getInput("github_token");
+    const debug = core.getInput("debug") === "true" || false;
+
+    if (debug) {
+        console.log(`Event name: ${github.context.eventName}`);
+        console.log(
+            `Event payload: ${JSON.stringify(
+                github.context.payload,
+                undefined,
+                2
+            )}`
+        );
+    }
+
+    const octokit = new Octokit({ auth: githubToken });
+
+    if (github.context.eventName === "issue_comment") {
+        const body = github.context.payload.comment.body;
+        const authorAssociation =
+            github.context.payload.comment.author_association;
+        const pull_number = github.context.payload.issue.number;
+
+        const { owner, repo } = github.context.repo;
+
+        if (body.trim() !== "/rerun") {
+            // unrecognized command
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        if (!github.context.payload.issue.pull_request) {
+            // not a pull request
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        if (github.context.payload.issue.state !== "open") {
+            // not open
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        console.log(`evaluating action for PR#${pull_number}`);
+
+        if (
+            authorAssociation !== "COLLABORATOR" &&
+            authorAssociation !== "OWNER" &&
+            authorAssociation !== "MEMBER" &&
+            authorAssociation !== "CONTRIBUTOR"
+        ) {
+            console.log(
+                `skipped due to authorAssociation: ${authorAssociation}`
+            );
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        const pull_request = unwrapResult(
+            await octokit.request(
+                "GET /repos/{owner}/{repo}/pulls/{pull_number}",
+                {
+                    owner: github.context.payload.repository.owner.login,
+                    repo: github.context.payload.repository.name,
+                    pull_number: pull_number,
+                }
+            )
+        );
+
+        if (debug) {
+            console.log(
+                `Pull request: ${JSON.stringify(pull_request, undefined, 2)}`
```

Comment:
The console can format objects for you.
```suggestion
                'Pull request:', pull_request
```